File: Web/siteCommerciale/modele/Traitement.php
<?php

class Traitement {
	
	public $id;
	public $nom;
	
	public function __construct()
	{
		$this->nom = "";
	}
	
	public function hydrateur($traitement)
	{
		$this->nom = $traitement->nom;
	}
}

File: AppliIHM/ihmSupervision/traitement.cpp
#include "traitement.h"
Traitement::Traitement()
{

}
Traitement::Traitement(std::string nomTraitement)
{
    this->nom = nomTraitement;
}



std::string Traitement::getNom()
{
    return this->nom;
}

void Traitement::setNom(std::string nom)
{

    this->nom = nom;
}



